tests/framework/CodeInterfaceTests/Simulate/test_GA-SIM3-loaddingpattern.xml

```diff
@@ -0,0 +1,367 @@
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc11">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc12">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc13">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc14">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc15">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc16">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc17">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc18">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc19">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc20">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc22">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc23">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc24">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc25">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc28">
+      <distribution>FA_dist</distribution>
+    </variable>
+    <variable name="loc29">
+      <distribution>FA_dist</distribution>
+	</variable>
+	<constant name="loc21">5.0</constant>
+	<constant name="loc26">5.0</constant>
+	<constant name="loc27">5.0</constant>
+	<constant name="loc30">5.0</constant>
+	<constant name="loc31">5.0</constant>
+	<constant name="loc32">5.0</constant>
+	<constant name="loc33">5.0</constant>
+	<constant name="loc34">5.0</constant>
+	<constant name="loc35">5.0</constant>
+    </MonteCarlo>
+  </Samplers>
+
+  <Steps>
+      <MultiRun name="sampleGA-Simulate" re-seeding="2286">
+      <Input class="Files" type="simulatedata">simulatedata_input</Input>
+      <Input class="Files" type="perturb">simulateperturb_input</Input>
+      <Input class="Files" type="input">input</Input>
+      <Model          class="Models" type="Code">MySimulate</Model>
+      <Optimizer      class="Optimizers"  type="GeneticAlgorithm" >GAopt</Optimizer>
+      <SolutionExport class="DataObjects" type="PointSet">opt_export</SolutionExport>
+      <Output class="DataObjects" type="PointSet">optOut</Output>
+      <Output class="OutStreams" type="Print">opt_export</Output>
+    </MultiRun>
+	<IOStep name="print">
+      <Input class="DataObjects" type="PointSet">opt_export</Input>
+      <Input class="DataObjects" type="PointSet">optOut</Input>
+      <Output class="OutStreams" type="Print">opt_export</Output>
+      <Output class="OutStreams" type="Print">optOut</Output>
+    </IOStep>
+  </Steps>
+  <Functions>
+    <External file="Simulate_RAVEN-GA/genmap.py" name="gen_pattern">
+      <variables>

```

Comment:
the path should be relative to the working directory. Please replace `Simulate_RAVEN-GA/` by `./`